ssl/statem/statem_srvr.c

```diff
@@ -3130,6 +3130,12 @@ static int tls_process_cke_dhe(SSL *s, PACKET *pkt)
         goto err;
     }
     cdh = EVP_PKEY_get0_DH(ckey);
+    if (cdh == NULL) {
+        SSLfatal(s, SSL_AD_INTERNAL_ERROR,
+                     SSL_F_TLS_PROCESS_CKE_DHE,
+                     ERR_R_INTERNAL_ERROR);
```

Comment:
That's not how we format things.
```
        SSLfatal(s, SSL_AD_INTERNAL_ERROR, SSL_F_TLS_PROCESS_CKE_DHE,
                 ERR_R_INTERNAL_ERROR);
```

Reply:
I just simply copied and modified the `SSLfatal` function in the npt checking condition in the same `.c` file. See [this commit](https://github.com/openssl/openssl/commit/a68236572850a1f50d5c40990b5a15a18ebea3bc).